x-pack/plugins/index_management/public/application/services/documentation.ts

```diff
@@ -299,6 +307,10 @@ class DocumentationService {
     return this.runtimeFields;
   }
 
+  public getBulkIdexAlias() {
```

Comment:
```suggestion
  public getBulkIndexAlias() {
```

Reply:
Hi, @sabarasaba thanks a lot for taking time to review the PR and test the changes. I have fixed the typo on my last commit. Have a look at your convenience.